### PR TITLE
SEO-1908-winforms-moneykeywords-syntax-editor-docs

### PR DESCRIPTION
**Pull request by Gowri-V-L**
Hi All,

Could you please review and merge the PR

### CODE REVIEW

File: WindowsForms/Syntax-Editor/Getting-Started.md [hunk 1]
```diff
@@ -260,3 +260,5 @@ Me.editControl1.ApplyConfiguration("LISP")
 {% endtabs %}
 
 ![Windows Forms EditControl configured for custom language](Getting-Started_images/Getting-Started_img5.png)
+
+N> You can refer to our [WinForms Syntax Editor](https://www.syncfusion.com/winforms-ui-controls/syntax-editor) feature tour page for its unique feature sets. You can also explore our [WinForms Syntax Editor example](https://github.com/syncfusion/winforms-demos/tree/master/edit) to know how to create interactive code editor applications with syntax highlighting, text indentation, intellisense, etc,.
```
Comment (Girija-Rajendran) [inline WindowsForms/Syntax-Editor/Getting-Started.md:264]: .. that shows how to create interactive code......

Comment (Gowri-V-L) [reply WindowsForms/Syntax-Editor/Getting-Started.md:264]: modified

**Comment by Gowri-V-L** [comment]
rebuild

**Comment by Gowri-V-L** [comment]
rebuild

**Review by Girija-Rajendran — APPROVED** [review]

**Review by ChristoIssac — APPROVED** [review]

**Pull request merged**